scrapers/errata.py

```diff
@@ -198,3 +216,34 @@ def get_attached_bugs(self, advisory_id):
         """.format(advisory_id)
         log.info('Getting Bugzilla bugs tied to the advisory with ID {0}'.format(advisory_id))
         return self.teiid.query(sql)
+
+    def get_product_version_info(self, product_id):
+        """
+        Query TEIID to find the product version name from a product ID.
+
+        :param int product_id: the product ID
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql = """\
+            SELECT pv.name as version_name, pv.id as product_version_id
+            FROM Errata_public.product_versions AS pv
+            WHERE pv.product_id = {0};
+        """.format(product_id)
+        return self.teiid.query(sql)
+
+    def get_associated_koji_tags(self, product_version_id):
+        """
+        Query TEIID to find the associated koji tag from a product version ID.
+
+        :param int product_version_id: the product version ID
+        :return: a list of dictionaries
+        :rtype: list
+        """
+        sql = """\
+            SELECT tags.id AS tag_id, tags.name AS tag_name
+            FROM Errata_public.brew_tags_product_versions AS tags_pv,
+                 Errata_public.brew_tags AS tags
```

Comment:
Could you make this an explicit join instead of using the implicit syntax for readability?